src/ontology/cl-edit.owl

```diff
@@ -8936,11 +8936,17 @@ SubClassOf(obo:CL_0000641 obo:CL_0000163)
 
 # Class: obo:CL_0000642 (folliculostellate cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "JB:jb") obo:IAO_0000115 obo:CL_0000642 "A supportive cell of the vertebrate pituitary that provides macromolecular transport and secretes hormones.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:18287078") Annotation(oboInOwl:hasDbXref "PMID:34550453") Annotation(oboInOwl:hasDbXref "PMID:34734454") Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") obo:IAO_0000115 obo:CL_0000642 "A supporting cell of the anterior pituitary (adenohypophysis). A folliculostellate cell can proliferate and differentiate into all pituitary endocrine cell types. This cell has been described as a small, agranular, star-shaped cell with long, slender processes.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") oboInOwl:hasExactSynonym obo:CL_0000642 "folliculo-stellate cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:34734454") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_0000642 "FS cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_0000642 "FSC")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:18287078") Annotation(oboInOwl:hasDbXref "PMID:34734454") Annotation(oboInOwl:hasDbXref "doi:10.23937/2572-407X.1510006") rdfs:comment obo:CL_0000642 "Folliculostellate cells have been shown to produce a range of growth factors and cytokines, such as vascular endothelial growth factor (VEGF), basic fibroblast growth factor, annexin 1 (ANXA1), Wnt ligands and interleukin-6, as well as nitric oxide (NO), all of which have clear roles in the regulation of specific endocrine pituitary cell types. This cell also acts as phagocyte of both cellular debris and apoptotic bodies. Folliculostellate cells can be identified by immunohistochemistry assays using the markers Sox2 and S100-beta.")
 AnnotationAssertion(rdfs:label obo:CL_0000642 "folliculostellate cell")
-SubClassOf(obo:CL_0000642 obo:CL_0000163)
```

Comment:
Consider ways to represent this cell type's role in the endocrine system.
Also consider if 'paracrine cell' would be an appropriate parent class.